packages/react-composites/src/composites/CallComposite/components/MediaGallery.tsx

```diff
@@ -118,3 +126,58 @@ export const useLocalVideoStartTrigger = (isLocalVideoAvailable: boolean, should
     }
   }, [shouldTransition, isButtonStatusSynced, isPreviewCameraOn, isLocalVideoAvailable, mediaGalleryHandlers]);
 };
+
+/**
+ * sets the announcement string whenever a Participant comes or goes from a call to be
+ * used by the system narrator.
+ *
+ * @returns string to be used by the narrator and Announcer component
+ */
+const useParticipantChangedAnnouncement = (): string => {
+  const adapter = useAdapter();
+  const locale = useLocale().strings.call;
+  const [announcerString, setAnnouncerString] = useState<string>('');
+  /**
+   * state to track whether there is currently a timer set in the MediaGallery
```

Comment:
nit: this checks if there is a timer set _in this hook_, not in the media gallery. The media gallery could have other timers unrealted to this hook

Reply:
ideally remove mention of the MediaGallery and move this to its own util file, it'll likely only be used here as we don't want this announcement happening all over the place but the implementation detail is not related to the MediaGallery so not necessary to bloat the file